TWLight/templates/header_partial_b3.html

```diff
@@ -104,7 +104,7 @@
             <li role="separator" class="divider"></li>
             {% if user and user.is_authenticated %}
               <li role="presentation" class="mobile-menu nav-item">
-                <a class="btn user-button" href="{% url 'users:home' %}">
+                <a class="btn twl-btn" href="{% url 'users:home' %}">
```

Comment:
You forgot to add the `twl-btn` class to local.css.

Reply:
That's really weird. This is correct in my local code already. Something something caching?

Reply:
... I see I misread this. I intended to leave this as user-button, I'll go back and revert that change.